assets/js/modules/tagmanager/components/common/ContainerSelect.js

```diff
@@ -43,12 +43,16 @@ export default function ContainerSelect( {
 	value,
 	...props
 } ) {
-	const accounts = useSelect( ( select ) => select( STORE_NAME ).getAccounts() );
+	const { accounts, hasResolvedAccounts } = useSelect( ( select ) => ( {
+		accounts: select( STORE_NAME ).getAccounts(),
+		hasResolvedAccounts: select( STORE_NAME ).hasFinishedResolution( 'getAccounts' ),
+	} ) );
+
 	const accountID = useSelect( ( select ) => select( STORE_NAME ).getAccountID() );
 	const hasExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasExistingTag() );
-	const isLoadingContainers = useSelect( ( select ) => select( STORE_NAME ).isDoingGetContainers( accountID ) );
+	const hasResolvedContainers = useSelect( ( select ) => select( STORE_NAME ).hasFinishedResolution( 'getContainers', [ accountID ] ) );
 
-	if ( accounts === undefined || containers === undefined || isLoadingContainers ) {
+	if ( accounts === undefined || ! hasResolvedAccounts || containers === undefined || ! hasResolvedContainers ) {
```

Comment:
I think the `undefined` checks still need to be removed from here.

Reply:
Yep, updated.